### ISSUE TITLE
4.3.30 installing oldest allowed numpy

### ISSUE THREAD

**Issue by jreback** [conda/conda#6260]
```
(pandas) bash-3.2$ conda info
Current conda install:

               platform : osx-64
          conda version : 4.3.30
       conda is private : False
      conda-env version : 4.3.30
    conda-build version : 2.1.3
         python version : 3.5.2.final.0
       requests version : 2.12.4
       root environment : /Users/jreback/miniconda3  (writable)
    default environment : /Users/jreback/miniconda3/envs/pandas
       envs directories : /Users/jreback/miniconda3/envs
                          /Users/jreback/.conda/envs
          package cache : /Users/jreback/miniconda3/pkgs
                          /Users/jreback/.conda/pkgs
           channel URLs : https://repo.continuum.io/pkgs/main/osx-64
                          https://repo.continuum.io/pkgs/main/noarch
                          https://repo.continuum.io/pkgs/free/osx-64
                          https://repo.continuum.io/pkgs/free/noarch
                          https://repo.continuum.io/pkgs/r/osx-64
                          https://repo.continuum.io/pkgs/r/noarch
                          https://repo.continuum.io/pkgs/pro/osx-64
                          https://repo.continuum.io/pkgs/pro/noarch
                          https://conda.anaconda.org/conda-forge/osx-64
                          https://conda.anaconda.org/conda-forge/noarch
            config file : /Users/jreback/.condarc
             netrc file : None
           offline mode : False
             user-agent : conda/4.3.30 requests/2.12.4 CPython/3.5.2 Darwin/15.6.0 OSX/10.11.6    
                UID:GID : 502:20
```

installing numpy 1.9.3
```
(pandas) bash-3.2$ conda create -n foo pandas --dry
Fetching package metadata .............
Solving package specifications: .

Package plan for installation in environment /Users/jreback/miniconda3/envs/foo:

The following NEW packages will be INSTALLED:

    ca-certificates: 2017.08.26-ha1e5d58_0      defaults
    certifi:         2017.7.27.1-py36hd973bb6_0 defaults
    intel-openmp:    2018.0.0-h68bdfb3_7        defaults
    libcxx:          4.0.1-h579ed51_0           defaults
    libcxxabi:       4.0.1-hebd6815_0           defaults
    libedit:         3.1-hb4e282d_0             defaults
    libffi:          3.2.1-hd939716_3           defaults
    libgfortran:     3.0.1-h93005f0_2           defaults
    mkl:             2018.0.0-h5ef208c_6        defaults
    ncurses:         6.0-ha932d30_1             defaults
    numpy:           1.9.3-py36h60773e1_3       defaults
    openssl:         1.0.2l-h492a243_5          defaults
    pandas:          0.21.0-py36hd8e0503_0      defaults
    pip:             9.0.1-py36h1555ced_4       defaults
    python:          3.6.3-hc655967_3           defaults
    python-dateutil: 2.6.1-py36h86d2abb_1       defaults
    pytz:            2017.2-py36h2e7dfbc_1      defaults
    readline:        7.0-h81b24a6_3             defaults
    setuptools:      36.5.0-py36h2134326_0      defaults
    six:             1.11.0-py36h0e22d5e_1      defaults
    sqlite:          3.20.1-h900c3b0_1          defaults
    tk:              8.6.7-hcdce994_1           defaults
    wheel:           0.29.0-py36h3597b6d_1      defaults
    xz:              5.2.3-ha24016e_1           defaults
    zlib:            1.2.11-h60db283_1          defaults
```

with a pin this is ok
```
(pandas) bash-3.2$ conda create -n foo pandas numpy=1.13 --dry
Fetching package metadata .............
Solving package specifications: .

Package plan for installation in environment /Users/jreback/miniconda3/envs/foo:

The following NEW packages will be INSTALLED:

    ca-certificates: 2017.08.26-ha1e5d58_0      defaults
    certifi:         2017.7.27.1-py36hd973bb6_0 defaults
    intel-openmp:    2018.0.0-h68bdfb3_7        defaults
    libcxx:          4.0.1-h579ed51_0           defaults
    libcxxabi:       4.0.1-hebd6815_0           defaults
    libedit:         3.1-hb4e282d_0             defaults
    libffi:          3.2.1-hd939716_3           defaults
    libgfortran:     3.0.1-h93005f0_2           defaults
    mkl:             2018.0.0-h5ef208c_6        defaults
    ncurses:         6.0-ha932d30_1             defaults
    numpy:           1.13.3-py36h2cdce51_0      defaults
    openssl:         1.0.2l-h492a243_5          defaults
    pandas:          0.20.3-py36hd6655d8_2      defaults
    pip:             9.0.1-py36h1555ced_4       defaults
    python:          3.6.3-hc655967_3           defaults
    python-dateutil: 2.6.1-py36h86d2abb_1       defaults
    pytz:            2017.2-py36h2e7dfbc_1      defaults
    readline:        7.0-h81b24a6_3             defaults
    setuptools:      36.5.0-py36h2134326_0      defaults
    six:             1.11.0-py36h0e22d5e_1      defaults
    sqlite:          3.20.1-h900c3b0_1          defaults
    tk:              8.6.7-hcdce994_1           defaults
    wheel:           0.29.0-py36h3597b6d_1      defaults
    xz:              5.2.3-ha24016e_1           defaults
    zlib:            1.2.11-h60db283_1          defaults

```

**Comment by jreback** [comment]
cc @kalefranz @msarahan

**Comment by mbargull** [comment]
This is a packaging issue. https://github.com/AnacondaRecipes/pandas-feedstock/pull/2 might be a fix for that.

**Comment by msarahan** [comment]
Fixed with build number 1 of the pandas 0.21.0 package.

**Comment by jreback** [comment]
thanks @msarahan
